### DISCUSSION TITLE
Error with setting up project with Firebase

### DISCUSSION THREAD

**Discussion by rishabh-kumar1** [AntlerVC/firetable#1113] [Support / Q&A] [answered]
Started the creation process for a project through Rowy, deleted it to restart due to inactivity (had to switch to other work), and after setting up the same project again, an error code appears. 

"Oops, something went wrong
This issue has been reported to the Rowy team to be fixed.

ROWY TRACING ID: ROWYERR-598B-FEE0-E0EA-A2385739D27C" 

Seems like a pretty large bug for this product.

**Comment by BeeBombshell** [comment]
Hey @rishabh-kumar1, I have taken this issue up with the team. Somebody will get back to you with a solution soon. 
Cc @shamsmosowi @bofeiw

**Comment by ancs21** [comment]
Hi @BeeBombshell , I also get the same issue:
```
ROWY TRACING ID: ROWYERR-31A0-86BA-1199-934ED0C1C5B2
```

**Comment by ghost** [comment]
I'm getting the same issue @BeeBombshell: `ROWY TRACING ID: ROWYERR-4A24-E0B2-189C-D0E23D290E51`

**Comment by BeeBombshell** [comment]
Hey @rishabh-kumar1, @ancs21, @avishaib-cc, thank you for bringing this up. We are still investigating this issue and will soon come up with a workaround for the same. Appreciate your patience!

**Comment by BeeBombshell** [comment, marked as answer]
Hey @rishabh-kumar1, @ancs21. Thank you for your patience, we're back with a solution that you can try on your end! I've created a page in our documentation that contains steps that you need to perform: https://docs.rowy.io/faqs/deleted-project. 
For the 4th step, you can drop your **Account email**, and **Project ID** here, or drop a mail on support@rowy.io, and we can reconfigure it on our end.

**Comment by bofeiw** [comment]
We added a self-service "disconnect" button, to connect an existing project, you can disconnect it during project setup.

![image](https://github.com/rowyio/rowy/assets/34177142/53dd4e1d-c43e-4be8-8842-bc81bd418d13)
